src/vs/workbench/contrib/search/browser/searchActionsNav.ts

```diff
@@ -42,13 +42,13 @@ registerAction2(class ToggleQueryDetailsAction extends Action2 {
 			},
 		});
 	}
-	run(accessor: ServicesAccessor) {
+	run(accessor: ServicesAccessor, ...args: any[]) {
 		const contextService = accessor.get(IContextKeyService).getContext(document.activeElement);
 		if (contextService.getValue(SearchEditorConstants.InSearchEditor.serialize())) {
-			(accessor.get(IEditorService).activeEditorPane as SearchEditor).toggleQueryDetails();
+			(accessor.get(IEditorService).activeEditorPane as SearchEditor).toggleQueryDetails(args.length > 0 ? args[0].show : undefined);
```

Comment:
I'm thinking either the comparison could be `args.length === 1` or that one can just pass in `args[0]?.show` instead and avoid the ternary that way.

Reply:
Or may be instead of ternary operator we can directly do like this
args?.[0]?.show